tools/gcms2isocor/gcms2isocor.xml

```diff
@@ -0,0 +1,31 @@
+<tool id="gcms2isocor" name="Conversion GCMS PostRun Analysis to IsoCor" version="0.1.0" profile="21.05">
+    <requirements>
+        <requirement type="package" version="8.0.121">openjdk</requirement> 
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       #set $ins = ""
+       #for $input in $input_gcms_files
+         #if $input
+           #set $ins = $ins + " " + str($input)
+         #end if
+        #end for
+
+        java -jar '$__tool_directory__/P2M2WorkflowsTools-assembly-0.1.jar' '$ins' --out '$output_isocor'
```

Comment:
Not sure, but I *think* [this error](https://github.com/genouest/galaxy-tools/pull/19/checks?check_run_id=3698619666#step:7:593) can be fixed like this:
```suggestion
        java -jar '$__tool_directory__/P2M2WorkflowsTools-assembly-0.1.jar' $ins --out '$output_isocor'
```

Reply:
done!